src/view.rs

```diff
@@ -54,27 +51,23 @@ pub fn Editor() -> impl IntoView {
             class="textentry"
             contenteditable
             spellcheck="false"
-            on:beforeinput=move |ev| { set_editor.write_untracked().handle_input(ev) }
-            on:mousedown=move |_| { set_editor.write_untracked().rationalize_selection() }
+            on:beforeinput=move |ev| { set_editor.write().handle_input(ev) }
```

Comment:
q: I'm not entirely clear on the significance of changing these all from their`untracked` forms